apex/provider/resource_clones_unmap_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://mozilla.org/MPL/2.0/
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	. "github.com/bytedance/mockey"
+	"github.com/dell/terraform-provider-apex/apex/helper"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccClonesUnmapResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: ProviderConfig + clonesUnmapResourceConfig,
+				Check:  resource.ComposeAggregateTestCheckFunc(),
+			},
+		},
+	})
+}
+
+func TestAccClonesUnmapResourceError(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				PreConfig: func() {
```

Comment:
@nikitajoshi1 can you add some documentation of what you are verifying?

Reply:
Added comments.